### PR TITLE
I changed the player_name field. My first pull request

### PR DESCRIPTION
**Pull request by silvaPool**

### CODE REVIEW

File: git-players.text [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <<record
-player_name: THS
+player_name: MTS
```
Comment (thspower) [inline git-players.text]: Hey can you add a new, second record? Like this

```
<<record
player_name: new name
player_display_name: new display name
player_github: etc
player_favorite_thing: etc
player_public_email: etc
```

Comment (silvaPool) [reply git-players.text]: Done!

Comment (thspower) [reply git-players.text]: "Add a player record to the table"

```
<<record
player_name: THS
player_display_name: THSPower
player_github: https://github.com/thspower
player_favorite_thing: Saving the world
player_public_email: hspowersport@gmail.com
<<record
player_name: MTS
player_display_name: MTSEnergy
player_github: https://github.com/thspower
player_favorite_thing: The pursuit of excellence
player_public_email: silva.friends22@gmail.com
```

Comment (silvaPool) [reply git-players.text]: New record added: Player's favorite game.

Comment (thspower) [reply git-players.text]: Like this

https://github.com/thspower/free-energy/pull/2/files

😅 Add a second player to the file

Comment (silvaPool) [reply git-players.text]: Now yes lol

File: git-players.text [hunk 2]
```diff
@@ -1,6 +1,13 @@
 <<record
-player_name: THS
-player_display_name: THSPower
+player_name: MTS
+player_display_name: MTSEnergy
 player_github: https://github.com/thspower
-player_favorite_thing: Saving the world
```
Comment (thspower) [inline git-players.text:5]: Hey I dont think the players already in the file should be changed. Only add a new player, don't change one.

Comment (silvaPool) [reply git-players.text:5]: I came back with the player who had been removed!

**Review by thspower — APPROVED** [review]

**Comment by thspower** [comment]
LGTM, merge when ready!

**Comment by silvaPool** [comment]
Done!

**Pull request merged**